packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -143,14 +179,14 @@ export function LineChart(props: LineChartProps): ReactElement | null {
     return (
         <View style={style.container}>
             {dataTypesResult instanceof Error ? (
-                <Text>{dataTypesResult.message}</Text>
+                <Text style={style.errorMessage}>{dataTypesResult.message}</Text>
             ) : (
                 <View style={style.chart}>
-                    <View style={style.gridAndLabelsRow}>
+                    <View style={{ flex: 1 }}>
```

Comment:
Do we need this one as fixed style? Same for line 189

Reply:
Yes, mx devs don't need to have access to these views for their designs. It creates confusion when we expose all of them, because finding a descriptive name is very hard.

Reply:
👍